docs/submit/osg-flock.md

```diff
@@ -0,0 +1,187 @@
+
+```console
+root@host # yum install osg-flock
+```
+
+Gratia Probe Configuration
+--------------------------
+
+Edit the `/etc/gratia/condor/ProbeConfig` file. Change the probe section to be:
+
+```file
+    ProbeName="condor:YOUR_HOSTNAME_HERE"
+    SiteName="YOUR_SITE_HERE"
+    Grid="OSG"
+    SuppressUnknownVORecords="0"
+    SuppressNoDNRecords="0"
+    SuppressGridLocalRecords="1"
+    QuarantineUnknownVORecords="1"
+    EnableProbe="1"
+
+    MapUnknownToGroup="1"
+    MapGroupToRole="1"
+    VOOverride="osg"
+```
+
+Please remember to enable and start the probe:
+
+```console
+root@host # systemctl enable gratia-probes-cron
+root@host # systemctl start gratia-probes-cron
+```
+
+GSI: HTCondor Configuration
+---------------------------
+
+The following is a starting point for the HTCondor configuration. Depending on other
+site configuration, some changes might be required. Put this in for example
+`/etc/condor/config.d/80-osg-flocking.conf`
+
+```file
+    #--  With glideins, there is nothing shared
+    CONDOR_HOST=$(FULL_HOSTNAME)
+    UID_DOMAIN=$(FULL_HOSTNAME)
+    FILESYSTEM_DOMAIN=$(FULL_HOSTNAME)
+    
+    #-- Authentication settings
+    SEC_DEFAULT_AUTHENTICATION = REQUIRED
+    SEC_DEFAULT_AUTHENTICATION_METHODS = FS,GSI
+    SEC_READ_AUTHENTICATION    = OPTIONAL
+    SEC_CLIENT_AUTHENTICATION  = OPTIONAL
+    SEC_ENABLE_MATCH_PASSWORD_AUTHENTICATION = TRUE
+    DENY_WRITE         = anonymous@*
+    DENY_ADMINISTRATOR = anonymous@*
+    DENY_DAEMON        = anonymous@*
+    DENY_NEGOTIATOR    = anonymous@*
+    DENY_CLIENT        = anonymous@*
+    
+    #--  Privacy settings
+    SEC_DEFAULT_ENCRYPTION = OPTIONAL
+    SEC_DEFAULT_INTEGRITY = REQUIRED
+    SEC_READ_INTEGRITY = OPTIONAL
+    SEC_CLIENT_INTEGRITY = OPTIONAL
+    SEC_READ_ENCRYPTION = OPTIONAL
+    SEC_CLIENT_ENCRYPTION = OPTIONAL
+    
+    #-- With strong security, do not use IP based controls
+    HOSTALLOW_WRITE = *
+
+    # OSG VO flocking host
+    OSG_FLOCK = flock.opensciencegrid.org
+    OSG_FLOCK_DN = /DC=org/DC=incommon/C=US/postalCode=53706/ST=WI/L=Madison/street=1210 West Dayton Street/O=University of Wisconsin-Madison/OU=OCIS/CN=flock.opensciencegrid.org
+
+    # submitters/campus factories to flock to
+    FLOCK_TO = $(FLOCK_TO) $(OSG_FLOCK)
+
+    # Who to trust?  Inlucde the <VO>_DN to trust the submitter
+    GSI_DAEMON_NAME = $(GSI_DAEMON_NAME), $(OSG_FLOCK_DN)
+
+    # This host's certificates
+    # These are needed if you are flocking to a glideinwms host
+    GSI_DAEMON_CERT = /etc/grid-security/hostcert.pem
+    GSI_DAEMON_KEY = /etc/grid-security/hostkey.pem
+```
+
+GSI: Requesting a Host Certificate
+---------------------------------
+
+A host certificate is used for authenticating your submit host to the OSG
+infrastructure. If you do not already have a certificate, you can request one
+using [these instructions](/security/host-certs/)
+
+Pool Password: HTCondor Configuration
+-------------------------------------
```

Comment:
@rynge do we actually still recommend that people use pool password? How does this work with the default GSI security that we ship in `osg-flock`?

Reply:
Pool password is an alternative, we can be used if GSI is difficult to set up for the admins.